### DISCUSSION TITLE
Custom queries usage page and full name display at select

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#9563] [Ideas]
During the process of creating my application I have created lots of Mysql queries. Some of those are in the end not used or I have duplicated some due to the fact that there is not an easy way to find where a query is used. 
I wonder if there is such a possibility or if not i think this would be a nice addition.

Also when working on pages the drop down that you select query does not display the full length. This is a problem for similarly named queries. So at least a display on hover should exist. Same happens at backend data page where you select a query to edit from left column.

**Comment by melohagan** [comment]
Hey @gmentzik 

These are great ideas - would definitely be nice additions. I've created a feature request from this discussion: https://github.com/Budibase/budibase/issues/9686
